crates/diesel_models/src/cards_info.rs

```diff
@@ -21,3 +29,23 @@ pub struct CardInfo {
     pub last_updated: Option<PrimitiveDateTime>,
     pub last_updated_provider: Option<String>,
 }
+
+impl ApiEventMetric for CardInfo {}
```

Comment:
Is `CardInfo` and `UpdateCardInfo` are API models?

If these are API models we need to create separate types in API Models and implement `ApiEventMetric` there and not in diesel_models